ckanext/geocodejob/plugin.py

```diff
@@ -168,7 +175,7 @@ def geocode_dataset(res_id):
                     current_record['electionDistrict'] = response.get('electionDistrict','')
                     new_records.append(current_record)
 
-                time.sleep(1)
+                time.sleep(2)
```

Comment:
Let's make it sleep random milliseconds. Between 0.5 and 1.5 seconds. time.sleep(randint(500,1500) / 1000)